src/item.cpp

```diff
@@ -1815,23 +1815,24 @@ void item::on_wield( player &p  )
     if( &p == &g->u && art != nullptr ) {
         g->add_artifact_messages( art->effects_wielded );
     }
+    std::string msg;
+    if (has_flag("SLOW_WIELD") && (! is_gunmod())) {
+        int d = 32; // arbitrary linear scaling factor
+        if      (is_gun())  d /= std::max((int) p.skillLevel(gun_skill()),  1);
+        else if (is_weap()) d /= std::max((int) p.skillLevel(weap_skill()), 1);
 
-     if (has_flag("SLOW_WIELD") && (! is_gunmod())) {
-         int d = 32; // arbitrary linear scaling factor
-         if      (is_gun())  d /= std::max((int) p.skillLevel(gun_skill()),  1);
-         else if (is_weap()) d /= std::max((int) p.skillLevel(weap_skill()), 1);
-
-         int const penalty = get_var("volume", (int) type->volume) * d;
-         if (penalty > 50) {
-             std::string msg;
-             if      (penalty > 250) msg = _("It takes you much longer than usual to wield your %s.");
-             else if (penalty > 100) msg = _("It takes you longer than usual to wield your %s.");
-             else                    msg = _("It takes you slightly longer than usual to wield your %s.");
-
-             p.add_msg_if_player(msg.c_str(), tname().c_str());
-             p.moves -= penalty;
-         }
-     }
+        int const penalty = get_var("volume", (int) type->volume) * d;
+        if (penalty > 50) {
+            if      (penalty > 250) msg = _("It takes you much longer than usual to wield your %s.");
+            else if (penalty > 100) msg = _("It takes you longer than usual to wield your %s.");
+            else                    msg = _("It takes you slightly longer than usual to wield your %s.");
+
+            p.add_msg_if_player(msg.c_str(), tname().c_str());
+            p.moves -= penalty;
+        }
+    }
+    msg = _("You wield your $s.");
+    p.add_msg_if_player(msg.c_str(), tname().c_str());
```

Comment:
You didn't need to move the `msg` variable out of the block, instead you can put the literal string directly into the `add_msg` call:
```C++
p.add_msg_if_player( _("..."), tname().c_str() );
```

And the string should contain a '%s', not a '$s' to be replaced with the item name.